rasa_core_sdk/forms.py

```diff
@@ -439,7 +463,25 @@ def _activate_if_required(self, tracker):
             return []
         else:
             logger.debug("Activated the form '{}'".format(self.name()))
-            return [Form(self.name())]
+
+            # collect values of required slots filled before activation
+            prefilled_slots = {}
+            for slot_name in self.required_slots(tracker):
+                if not self._should_request_slot(tracker, slot_name):
+                    prefilled_slots[slot_name] = tracker.get_slot(slot_name)
+
+            if prefilled_slots:
+                logger.debug(
+                    "Validating pre-filled required slots: {}".format(prefilled_slots)
+                )
+                validation_events = self.validate_slots(
```

Comment:
let's just call it events like everywhere
```suggestion
                events.extend(self.validate_slots(
```